frontend/public/components/storage/create-pvc.tsx

```diff
@@ -197,6 +197,9 @@ export const CreatePVCForm: React.FC<CreatePVCFormProps> = (props) => {
     setAccessMode(event.currentTarget.value);
   };
 
+  //Accept anything except noobaa provisioned
+  const filterStorageClasses = (sc) => !sc.provisioner.includes('noobaa.io/obc');
```

Comment:
I dont think we should have noobaa specific stuff in core console. Shouldn't this be done via some extension point ?

Reply:
actually there's already a ceph specific stuff which just got moved to `storage-utils` see https://github.com/openshift/console/pull/3312 . You could use a similar approach